src/olympia/signing/tests/test_views.py

```diff
@@ -64,18 +66,18 @@ def create_version(self, version):
         response = self.request('PUT', self.url(self.guid, version), version)
         assert response.status_code in [201, 202]
 
-    def xpi_filepath(self, addon, version):
+    def xpi_filepath(self, guid, version):
```

Comment:
sorry for the noise here... the guid param being called `addon` was an itch I had to scratch